lib/glific_web/resolvers/partners.ex

```diff
@@ -343,4 +344,28 @@ defmodule GlificWeb.Resolvers.Partners do
       {:ok, %{credential: updated_credential}}
     end
   end
+
+  @doc """
+  Gets daily app usage
+  """
+  @spec get_app_usage(Absinthe.Resolution.t(), %{from_date: Date.t(), to_date: Date.t()}, %{
+    context: map()
+    }) ::
+    {:ok, list(map())} | {:error, String.t()}
+  def get_app_usage(_parent, %{from_date: from_date, to_date: to_date}, %{context: %{current_user: user}}) do
+    case PartnerAPI.get_app_usage(user.organization_id, Date.to_string(from_date), Date.to_string(to_date)) do
+      {:ok, response} -> {:ok, Enum.map(response, fn day ->
+        %{
+          date: Map.get(day, "date", 0),
+          incoming_msg: Map.get(day, "incomingMsg", 0),
+          outgoing_media_msg: Map.get(day, "outgoingMediaMsg", 0),
+          outgoing_msg: Map.get(day, "outgoingMsg", 0),
+          total_fees: Map.get(day, "totalFees", 0),
+          total_msg: Map.get(day, "totalMsg", 0)
```

Comment:
can u add a single line comment in code what each metric mean and also add other metrics as well